<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Message Windows</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>111</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>QMessageBox Icons</string>
    </property>
    <widget class="QPushButton" name="btnMessageBoxCritical">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Critical</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMessageBoxInformation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Information</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMessageBoxWarning">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Warning</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMessageBoxQuestion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Question</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnMessageBoxDetailedText">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Detailed Text</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>QMessageBox Icons Short</string>
    </property>
    <widget class="QPushButton" name="btnMessageBoxCriticalShort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Critical</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMessageBoxInformationShort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Information</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMessageBoxWarningShort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Warning</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMessageBoxQuestionShort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Question</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnMessageBoxAllButtons">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Alle Buttons</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMessageBoxTest">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok, Abbrechen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
